scripts/docker/start.sh

```diff
@@ -13,7 +13,8 @@ mkdir -p tmp/pids/
 rm -f tmp/pids/*.pid
 
 if [ "$RAILS_ENV" == "development" ]; then
-  exec foreman start
+  bundle check || bundle install --binstubs="$BUNDLE_BIN"
```

Comment:
Should this also be done for running the test suite?

Reply:
for sure - nice catch :)

Reply:
right - so the test suite won't run this entry point. The gems will have to be installed manually via a docker run command. i've updated the readme's accordingly.